lib/Echidna/Exec.hs

```diff
@@ -121,11 +124,25 @@ fuzz l n ts v p = do
     where run cs = p $ execState (forM_ cs execCall) v
 
 
-checkETest :: VM -> Text -> Bool
-checkETest v t = case evalState (execCall (t, [])) v of
-  VMSuccess (B s) -> s == encodeAbiValue (AbiBool True)
+type TestFunction = VM -> Text -> Bool
```

Comment:
I don't know if I believe this alias makes things clearer, especially if we merge these functions as suggested above

Reply:
This alias was created to avoid exporting VM and Bool into the Main module. If you don't care about them, I can remove it.